### PR TITLE
Feature/friendly guide for users

### PR DESCRIPTION
**Pull request by minsulee2**
Updates:
- New README!
- Coding convention
- to v1.1.7

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ainblockchain/ain-util",
-  "version": "1.1.6",
+  "version": "1.1.7",
```
Comment (liayoo) [inline package.json:3]: do we need a new version for the changes?

**Review by liayoo — APPROVED** [review]
LGTM with a question. Thanks for the updates! 🙌

File: package.json (hunk 1 above)
Comment (minsulee2) [reply package.json:3]: As version up was not made in the prev updates, I updated it now. Thanks for the question!

**Comment by minsulee2** [comment]
Thanks for the fast review! merging now :)

**Pull request merged**